packages/data-explorer/tests/unit/components/dataView/CustomEntityDetail.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { shallowMount } from '@vue/test-utils'
+import CustomEntityDetail from '@/components/dataView/CustomEntityDetail.vue'
+
+describe('CustomEntityDetail.vue', () => {
+  const propsData = {
+    record: {
+      'col1': 'Column 1',
+      'col2': 'Column 2',
+      'col3': 'Column 3'
+    },
+    metaData: {
+      meta1: 'meta 1',
+      meta2: 'meta 2'
+    },
+    template: '    <h1>Hello MOLGENIS!</h1>    '
+  }
+  const wrapper = shallowMount(CustomEntityDetail, { propsData })
+  it('exists', () => {
+    expect(wrapper.exists()).toBeTruthy()
+  })
+  it('should set the template', () => {
```

Comment:
should it also strip whitespace? Your template obviously has whitespace padding left and right.